icu4c/source/common/unicode/urename.h

```diff
@@ -109,10 +109,20 @@
 #define _UTF32LEData U_ICU_ENTRY_POINT_RENAME(_UTF32LEData)
 #define _UTF7Data U_ICU_ENTRY_POINT_RENAME(_UTF7Data)
 #define _UTF8Data U_ICU_ENTRY_POINT_RENAME(_UTF8Data)
+#define __bss_start U_ICU_ENTRY_POINT_RENAME(__bss_start)
```

Comment:
_bss_start is a system symbol, not an ICU one. I don't think it should be renamed.

Reply:
Fixed (this and a few others (as discussed).